src/components/chip-set/chip-set.tsx

```diff
@@ -577,6 +578,7 @@ export class ChipSet {
                         {chip.text}
                     </span>
                 </span>
+                {chip.badgeValue ? <limel-badge label={chip.badgeValue} /> : ''}
```

Comment:
I'm not a big fan of inline rendering conditions like this using the ternary operator since it can make the code harder to read. I suggest extracting this to a separate method instead, e.g. `renderBadge`